### ISSUE TITLE
VoiceOver not reading multiselects well

### ISSUE THREAD

**Issue by claudiulodro** [woocommerce/selectWoo#7]
> Multiselect: In a search, VoiceOver does not announce the option found. When an option is selected (it looks like a tag), VoiceOver announce it as a group. Even if the option is selected it, I was not able to delete it. I tried using space bar and the Enter key.

- Multiselect elements are announced as a "group".
- ctrl + space deletes elements, but this may be a non-standard hotkey that people don't know about

**Comment by bizcook** [comment]
Is there a fix to this yet? or a patch to get the options to be read aloud when navigating through them?

**Comment by danielck** [comment]
I came here hopeful that selectWoo would offer some help to a website built using select2, but alas I can second the issue reported by @claudiulodro that navigating with VoiceOver (testing on macOS) is very confusing. "Group" does not really convey that the tags are tags.  Ctrl-delete also deletes elements (actually changes them back into editable text).

Currently the whole field is defined as an aria-live region, which means that the changing content is repeatedly read aloud in it's entirety. For instance, after typing in "Albania", then started to write "france" but deleted and changed it to "Finland", then "South Africa", VoiceOver first announced:

"Albania, Finland, South Africa, collapsed, group"

and immediately after that

"Albania, fr, South Africa".

The latter was an intermediate version of what I had typed, but was announced later presumably because of the handling of aria-live=polite.

Perhaps consider not making the whole field an aria-live region but instead having a separate visually hidden aria-live region for announcing only successful additions?
